src/components/VideoPlayer/VideoPlayer.stories.mdx

```diff
@@ -0,0 +1,79 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import VideoPlayer from "./VideoPlayer";
+import { VideoPlayerTypes } from "./VideoPlayerTypes";
+import { action } from "@storybook/addon-actions";
+
+<Meta
+  title="VideoPlayer"
+  component={VideoPlayer}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10968%3A3680",
+    },
+  }}
+  argTypes={{
+    attributes: { table: { disable: true } },
+    className: { table: { disable: true } },
+    modifiers: { table: { disable: true } },
+  }}
+/>
+
+# Video Player
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.23.2`   |
+
+<Description of={VideoPlayer} />
+
+The `VideoPlayer` component is used to embed a `Vimeo` or `YouTube` video player on a page. The component requires both the `videoType` and `videoId` props. You can extract the videoId from the video's URL.
+
+<Preview withToolbar>
+  <Story
+    name="VideoPlayer"
+    args={{
+      videoType: VideoPlayerTypes.Vimeo,
+      videoId: "474719268",
+    }}
+  >
+    {args => (
+      <VideoPlayer
+        {...args}
+        videoId={args.videoType === "vimeo" ? "474719268" : "nm-dD2tx6bk"}
+      />
+    )}
+  </Story>
+</Preview>
+
+<ArgsTable story="VideoPlayer" />
+
+**Example Vimeo IDs:** `474719268`, `491405018`, `493795778` (square)<br />
```

Comment:
What does the "(square)" here mean? To use with the square aspect ratio? I don't see a difference for this video in this aspect ratio.

Reply:
Yes, that video is square.  And I just realized that the videoId input in the argsTable does not override the videoId that is being set in the code.  Let me see what I can do about that.

Reply:
There was a bug.  I fixed it.